src/tests/sqlsmith/tests/testdata/time_window.sql

```diff
@@ -0,0 +1 @@
+create table t1 (row_id int, uid int, v int, created_at timestamp);
```

Comment:
We can copy the nextmark schema here such that it can produce more time-related queries.